composer/algorithms/low_precision_groupnorm/low_precision_groupnorm.py

```diff
@@ -113,6 +105,19 @@ def _cast_if_autocast_enabled(tensor):
 
 
 def _to_LPGroupNorm(layer: torch.nn.Module, module_index: int) -> LPGroupNorm:
+    """Defines a replacement policy from a `torch.nn.GroupNorm` to a `LPGroupNorm`"""
     if not isinstance(layer, torch.nn.GroupNorm):
         raise TypeError(f'Expected torch.nn.GroupNorm, got {type(layer)}')
-    return LPGroupNorm(layer)
+    lp_groupnorm = LPGroupNorm(layer.num_groups, layer.num_channels, layer.eps, layer.affine)
+
+    with torch.no_grad():
+        if layer.weight is None:
+            lp_groupnorm.weight = None  # type: ignore
```

Comment:
Should this be:
```suggestion
            lp_groupnorm.register_parameter('weight', None )  # type: ignore
```
Not sure if ` # type: ignore` is needed in this case.
see [eg here](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/linear.py#L100)

same for bias, and layernorm weight and bias.